index.html

```diff
@@ -3,21 +3,237 @@
+        <div class="container">
+            <a class="navbar-brand d-none d-md-flex" href="./index.html"><img class="conta" src="img/work.png" alt="logo"><h1>Kadoma <br> Community Development</h1></a>
+            <div class="hamburger d-lg-none">
+                <img class="ham" src="img/ham2.png" alt="hamburger-menui">
+            </div>
+            <div class="d-none d-lg-block justify-content-end" id="navbarNav">
+                <div class="d-flex align-items-center justify-content-around">
+                    <a class="navbar-brand d-lg-none " href="./index.html"><img class="cont" src="img/work.png" alt="brand"><h1>Kadoma <br> Community Development</h1></a>
+                    <div class="close d-none">
+                        <i class="material-icons justify-content-end d-flex mx-3">close</i>
+
+                    </div>
+
+                </div>
+                <ul class="navbar-nav align-items-center">
+                    <li class="nav-item mx-2">
+                        <a class="nav-link" href="./index.html">Home</a>
+                    </li>
+                    <li class="nav-item mx-2">
+                        <a class="nav-link" href="./about.html">About</a>
+                    </li>
+                    <li class="nav-item mx-2">
+                        <a class="nav-link" href="#">Program</a>
+                    </li>
+                    <li class="nav-item mx-2">
+                        <a class="nav-link" href="#">Join</a>
+                    </li>
+                    <li class="nav-item mx-2">
+                        <a class="nav-link" href="#">Spoinsor</a>
+                    </li>
+                    <li class="nav-item mx-2">
+                        <a class="nav-link" href="#">News</a>
+                    </li>
+                    <li class="nav-item mx-2 border-nav">
+                        <a class="nav-link" href="#">Empowerment Indigenise</a>
+                    </li>
+
+                </ul>
+
+            </div>
+
+        </div>
+
+    </nav>
+
+    <div class="container-fluid intro-bg pb-5">
+        <section class="intro container pt-5">
+            <h5 class="hello-text">"Hello! Kadoma community"</h5>
+            <h2>KADOMA COMMUNITY<br> DEVELOPMENT COLLABORATIONS AND<br> WORK ALONGS</h2>
+            <div class="intro-d">
+                <p class="intro-details my-3 py-3">
+                    changing communities and wards through self development projects to empower the future generation wealth 
+                    to the communities and enriching lives through employment creation
+                </p>
+                <h2 class="date my-5">2021.10.27(MON)~30(WED)</h2>
+                <p class="location"> @Rimuka Stadium Ward 34 Council</p>
+            </div>
+        </section>
+    </div>
+    <div class="container-fluid what-we-learn">
+        <section class="container">
+            <div class="py-4">
+                <h3 class="text-center">Main Projects</h3>
```

Comment:
* Kindly fix this section as it is differnt for,m the original design 🙏 
* YOURS : 
![image](https://user-images.githubusercontent.com/52497246/133553661-1092a4cd-d808-4bba-803f-483bf6ad9c33.png)
* ORIGINAL : 
![image](https://user-images.githubusercontent.com/52497246/133553706-b89d77b5-d53d-48b4-a0b0-6fcae5083cc0.png)